pgd_core/templates/profile.html

```diff
@@ -5,6 +5,31 @@
 
 {% block content %}
 	<head> Welcome, {{full_name}} </head>
+
+	<div style="position:fixed; top:70px; right:90px; width:200px;">
+		<form method="post">{% csrf_token %}
+
+			{{ form.query.label }} <br>
+			{{ form.query}} <br>
+			{% for choice in form.search_type %}
+	  			{{ choice.choice_label }}
+	  			<span class="radio"> {{ choice.tag }} </span> <br>
+			{% endfor %}
+
+		</form>
+	</div> 
+
+<style>
```

Comment:
Isn't l:71 inside table ? was it inside the table before ? If so, i couldn't recall as to why i moved this up. And, yeah took a note of indentation. :)